lib/shopify-cli/oauth.rb

```diff
@@ -14,7 +14,7 @@ class OAuth
 
     class Error < StandardError; end
 
-    REDIRECT_HOST = "http://localhost"
+    REDIRECT_HOST = "http://app-cli-loopback.shopifyapps.com"
```

Comment:
Why HTTP rather than HTTPS?

Reply:
because it is listening locally. That domain is a loopback which the DNS resolves to 127.0.0.1 Setting up an https server on each person's computer is hard because we would need to distribute out our certs which makes it insecure anyways.

Reply:
TIL. I didn't realize that domain is on the loopback address. Thanks!

Reply:
Just that subdomain is. I set it up specifically for the cli